source/_docs/ecosystem/certificates/tls_self_signed_certificate.markdown

```diff
@@ -6,7 +6,12 @@ redirect_from: /cookbook/tls_self_signed_certificate/
 
 If your Home Assistant instance is only accessible from your local network you can still protect the communication between your browsers and the frontend with SSL/TLS. 
 [Let's encrypt]({{site_root}}/blog/2017/09/27/effortless-encryption-with-lets-encrypt-and-duckdns/) will only work if you have a DNS entry and remote access is allowed. 
-The solution is to use a self-signed certificate. As you most likely don't have a certification authority (CA) your browser will complain about the security. If you have a CA then this will not be an issue.
+
+The solution is to use a self-signed certificate. Please note however that after you have completed these steps, your browser will complain about the security of the certificate as it was not issued by a trusted authority.
+* This is due to self-signed certificates having not been issued by a certification authority ([`CA`](https://cheapsslsecurity.com/blog/what-is-a-certificate-authority-ca/)). If you have your own CA then this will not be an issue.
```

Comment:
there are MANY pages one could find/reference to explain what a CA is; I chose this one fairly arbitrarily, open to alternatives or preferences. (don't see what HA's policy is on acceptable external refs)

Reply:
That looks fine, thanks!